test_util/main.tf

```diff
@@ -50,15 +50,32 @@ variable "dcos_license_key_contents" {
   description = "Defines content of license used for EE."
 }
 
+variable "instance_type" {
+    type = "string"
+    default = "t3.medium"
+    description = "Defines type of used machine."
+}
+
+variable "build_id" {
+    type = "string"
+    default = ""
+    description = "Build ID from CI."
+}
+
+variable "build_type" {
+    type = "string"
+    default = ""
+    description = "Build type from CI."
+}
+
 # Used to determine your public IP for forwarding rules
 data "http" "whatismyip" {
   url = "http://whatismyip.akamai.com/"
 }
 
 resource "random_string" "password" {
-  length = 6
-  special = true
-  override_special = "-"
+  length = 12
```

Comment:
@janisz  - I think, we will run into problems here.

* Terraform / aws has limits on size of the string the characters allowed.

Could you please test this locally before using it?

Reply:
By testing locally, I meant, try to test it 3-5 times.

Reply:
Good point, I'll change to 6 and cluster ID so names will be shorter.

Reply:
Updated can you take another look

Reply:
Ouch, it was already on train. I'll fix it later but all runs local + 3 on CI passed so I think we do not need to change it.